grunt_tasks/webpack.config.js

```diff
@@ -74,6 +74,7 @@ module.exports = {
             // ES2015
             {
                 test: /\.js$/,
+                include: /clients\/web\/src/,
```

Comment:
With these restrictions, does it make it so that the built-in plugins don't use these rules? If so, that would explain why they are failing to build.